.github/workflows/cmake.yml

```diff
@@ -157,8 +157,12 @@ jobs:
         echo "::endgroup::"
         [ "y" = ${run_worked} ]
       timeout-minutes: 10
```

Comment:
Increasing the timeout will probably help :)

Reply:
Yeah.... building on my local machine again so I can `time ./run_tests_and_shutdown.sh` to see how long this is *actually* going to take without KVM... I suspect there's a few benchmarks that are ... aggressive.

Reply:
It looks like most of the time is spent running benchmarks in LibGfx?

Reply:
Filter out this guy? 

```
Sun, 18 Apr 2021 20:08:54 GMT ==== Running test 11 out of 88 -- /usr/Tests/LibGfx/painter ====
Sun, 18 Apr 2021 20:08:54 GMT /usr/Tests/LibGfx/painter
Sun, 18 Apr 2021 20:08:55 GMT  Running benchmark 'diagonal_lines'.
Sun, 18 Apr 2021 20:10:46 GMT  Running benchmark 'fill'.
Sun, 18 Apr 2021 20:11:04 GMT  Running benchmark 'fill_with_gradient'.
Sun, 18 Apr 2021 20:17:23 GMT  Running 3 cases out of 3.
```

Reply:
Jeez. Yeah. I updated to 20 mins (hopefully not necessary) and filtered out that painter benchmark.